### PR TITLE
fix: validate operation body

### PR DESCRIPTION
**Pull request by n1ru4l**
### Background

tba

### Description

tbd

### Checklist

- [ ] Input validation
- [ ] Output encoding
- [ ] Authentication management
- [ ] Session management
- [ ] Access control
- [ ] Cryptographic practices
- [ ] Error handling and logging
- [ ] Data protection
- [ ] Communication security
- [ ] System configuration
- [ ] Database security
- [ ] File management
- [ ] Memory management
- [ ] Testing

### CODE REVIEW

File: packages/services/usage/src/usage-processor-2.ts [hunk 1]
```diff
@@ -83,6 +84,10 @@ export function usageProcessorV2(
 
     let newOperationMapKey = newKeyMappings.get(operationMapRecord);
 
+    if (!isValidOperationBody(operationMapRecord.operation)) {
+      return null;
```
Comment (kamilkisiela) [inline packages/services/usage/src/usage-processor-2.ts:88]: `null` will lead to an incorrect information for us, I mean this metric: 
https://github.com/kamilkisiela/graphql-hive/blob/38a17d8f2fdd9ae1c8ae0622cdea156c1ccc45db/packages/services/usage/src/usage-processor-2.ts#L119

Comment (kamilkisiela) [reply packages/services/usage/src/usage-processor-2.ts:88]: Maybe we could do what we do in usage-1, return to the sender that N operations were accepted and M operations rejected?

Comment (n1ru4l) [reply packages/services/usage/src/usage-processor-2.ts:88]: @kamilkisiela we already do, see tests: [`7b01304` (#5537)](https://github.com/kamilkisiela/graphql-hive/pull/5537/commits/7b013041342f140c5ffdecdf8e6d026c7d3e1497)

Comment (n1ru4l) [reply packages/services/usage/src/usage-processor-2.ts:104]: fixed the incorrect information in [`d9a3e56` (#5537)](https://github.com/kamilkisiela/graphql-hive/pull/5537/commits/d9a3e56bc27d5e613884addbdfe5dfe23f30e51a)

**Pull request merged**